exo/main.py

```diff
@@ -59,9 +62,21 @@
 inference_engine = get_inference_engine(inference_engine_name, shard_downloader)
 print(f"Using inference engine: {inference_engine.__class__.__name__} with shard downloader: {shard_downloader.__class__.__name__}")
 
-if args.node_port is None:
-  args.node_port = find_available_port(args.node_host)
-  if DEBUG >= 1: print(f"Using available port: {args.node_port}")
+if args.discovery_module == "manual":
+  if args.discovery_config is None:
+    args.discovery_module == "udp"
+  else:
+    list_device = ReadManualConfig(discovery_config=args.discovery_config)
+    list_device.device_capabilities((str((list_device.whoami))))
```

Comment:
im confused why there's another function call here.

Reply:
There is 2 call fo ReadManualConfig.
One in the main for the current instance so the main.py is instancied with his own value.
The second one for the discovery of the other devices.